workflow/scripts/ice-floe-tracker.jl

```diff
@@ -21,27 +51,35 @@ function main(args)
         "extractfeatures"
         help = "Extract ice floe features from segmented floe image"
         action = :command
-    end
 
-    mkclipreprocess!(settings)
-    mkcliextract!(settings)
-    mkclitrack!(settings)
+        "track"
+        help = "Pair ice floes in day k with ice floes in day k+1"
+        action = :command
+    end
 
-    landmask_cloudmask_args = [
-        "input",
-        Dict(:help => "Input image directory", :required => true),
-        "output",
-        Dict(:help => "Output image directory", :required => true),
-    ]
+    command_common_args = [
+        "--log",
+        Dict(:help => "Show log on terminal; either 1 or 0", :required => false, :arg_type => Int,
+            :default => 0, :range_tester => (x -> x == 0 || x == 1)
+        )]
 
-    add_arg_table!(settings["landmask"], landmask_cloudmask_args...)
+    mkcli!(settings, command_common_args)
 
     parsed_args = parse_args(args, settings; as_symbols=true)
 
     command = parsed_args[:_COMMAND_]
     command_args = parsed_args[command]
     command_func = getfield(IFTPipeline, Symbol(command))
-    @time command_func(; command_args...)
+    logoption = command_args[:log]
```

Comment:
will the logger report back on the time for each task to run? I'm assuming it replaces the @time macro used before, but just wanted to c=double check!

Reply:
Good catch! That didn't merge right. Fixed!